guardian/support-admin-console
# Update to ticker config

## What does this change?
Removing some fields from the UI which are no longer required:
- Goal reached text 
- Unlimited end type option
- Adds in new ticker option on the banner colour options

[Related DCR PR](https://github.com/guardian/dotcom-rendering/pull/12153)



## How to test
Tested on CODE, create a new banner deisgn and select your ticker colurs. Create a new banner test under 'Banner 1' and configure a ticker. Check the ticker appears on the test banner on DCR (assuming it is up to date with the change).


## How can we measure success?



## Have we considered potential risks?



## Images
Before
![image](https://github.com/user-attachments/assets/27acbb65-0fdd-4203-b670-69b9e2d10fa9)

After
![image](https://github.com/user-attachments/assets/5a8f08aa-bae7-4cc5-ac96-cc97a8a1b740)

New fields added to colour picker with those circled in red being deprecated (which will be done in a follow up PR)
![image](https://github.com/user-attachments/assets/6be43cba-9e32-4e57-bbca-f5e244d15d2a)


## Accessibility
